examples/PrivateLink/aws.tf

```diff
@@ -1,19 +1,9 @@
-variable "aws_key" {
-  type = string
-}
-
-variable "aws_secret" {
-  type = string
-}
-
 variable "aws_region" {
   type = string
 }
 
 provider "aws" {
   region     = var.aws_region
-  access_key = var.aws_key
```

Comment:
How is this change related with this PR?

Reply:
Just modifying the example so it works from environment vars so it is easier to run